ntbs-service/Helpers/RouteHelper.cs

```diff
@@ -5,6 +5,7 @@ namespace ntbs_service.Helpers
     public static class RouteHelper
     {
         public static string NotificationBasePath => "/Notifications/{0}/{1}";
+        public static string AlertBasePath => "/Alerts/{0}/{1}";

```

Comment:
is this actually used anywhere externally? If not, perhaps it should just be a variable in the `GetAlertPath` method?